### PR TITLE
Fixed URL bug

### PR DESCRIPTION
**Pull request by maxneuvians**
Fix for #454

### CODE REVIEW

**Comment by sastels** [comment]
having problems in guided experience

File: pages/_app.js [hunk 1]
```diff
@@ -8,37 +8,82 @@ import { initStore } from "../store";
...
+      if (
+        ctx.query.statusAndVitals &&
+        ctx.query.statusAndVitals !== currentState.statusAndVitals
+      ) {
+        ctx.store.dispatch({
+          type: "SET_STATUS_TYPE",
+          data: ctx.query.statusAndVitals
+        });
+      }
+      if (!ctx.query.serviceType && currentState.serviceType !== "") {
```
Comment (smcmurtry) [inline pages/_app.js]: this is the same if statement as line 40?

**Comment by maxneuvians** [comment]
Good catches @sastels and @smcmurtry

**Review by sastels — APPROVED** [review]
Seems good now! 🦑

**Pull request merged**